state/presence/presence.go

```diff
@@ -79,6 +87,18 @@ func docIDStr(modelUUID string, localID string) string {
 
 // BUG(gn): The pings and beings collection currently grow without bound.
 
+// psuedoRandomFactor defines an increasing chance that we will trigger an effect.
+// Inspired by: http://dota2.gamepedia.com/Random_distribution
+// The idea is that 'on average' we will trigger 5% of the time. However, that
+// leaves a low but non-zero chance that we will *never* trigger, and a
+// surprisingly high chance that we will trigger twice in a row.
+// psuedoRandom increases the chance to trigger everytime it does not trigger,
+// ultimately making it mandatory that you will trigger, and giving the desirable
+// average case that you will trigger while still giving some slop so that
+// machines won't get into sync and trigger at the same time.
+// psuedoRandomFactor of 0.00380 represents a 5% average chance to trigger.
+const psuedoRandomFactor = 0.00380
```

Comment:
thanks. its interesting how it changes 'real random' for more 'how you expect random to feel'.